### PR TITLE
[Feat] Focus on amount input when clicking the blue square

### PR DESCRIPTION
**Pull request by leablock**
# Name of the feature/issue

Focus amount input when clicking on the div

### CODE REVIEW

File: packages/ui/src/routes/bridge/BridgeSetupPage.tsx [hunk 1]
```diff
@@ -556,6 +561,7 @@ const BridgeSetupPage: FunctionComponent<{}> = () => {
                         >
                             <input
                                 {...register("amount")}
```
Comment (fpezzola) [inline packages/ui/src/routes/bridge/BridgeSetupPage.tsx:563]: The `register` function already returns a ref and we're overriding it.

We can either use the `useMergeRef` hook or do this: 

```javascript
const { ref: amountInputRef, ...amountInputProps } = register("amount");

return (
<>
  ...
 <input  ref={amountInputRef} {...amountInputProps} id="amount" .../>
</>
)
```

Comment (leapalazzolo) [reply packages/ui/src/routes/bridge/BridgeSetupPage.tsx]: Good catch!

Comment (leablock) [reply packages/ui/src/routes/bridge/BridgeSetupPage.tsx]: Good catch!

**Review by fpezzola — APPROVED** [review]
LGTM!

**Pull request merged**